src/main/java/com/squarepolka/readyci/tasks/app/android/AndroidSignApp.java

```diff
@@ -29,6 +29,8 @@ public void performTask(BuildEnvironment buildEnvironment) throws TaskFailedExce
         String scheme = buildEnvironment.getProperty(BUILD_PROP_SCHEME);
         String keystorePath = buildEnvironment.getProperty(BUILD_PROP_JAVA_KEYSTORE_PATH);
         String storePass = buildEnvironment.getProperty(BUILD_PROP_STOREPASS);
+
+        // TODO: these are likely to be incorrect
```

Comment:
Do we need to clean this one up before we merge?

Reply:
should be a separate fix, this is just something while i was digging around